tensorflow/compiler/mlir/tools/kernel_gen/BUILD

```diff
@@ -135,6 +138,11 @@ tf_cc_binary(
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:TargetLLVMIR",
     ],
+    ] + if_llvm_system_z_available([
```

Comment:
This causes a syntax error. I think you want to remove line 140 (GitHub doesn't let me do an automatic suggestion and merge)

Reply:
Good catch. Not sure where that was missed, maybe by the rebase. Anyway, fixed